GitUI/Theming/Renderers/ThemeProperty.cs

```diff
@@ -0,0 +1,255 @@
+﻿namespace GitUI.Theming
+{
+    internal enum ThemeProperty
```

Comment:
What are these, and where did they come from?

Reply:
These are WinApi theming property identifiers, they are used in GetThemeColor.
Searchable by `public override int GetThemeColor`. Whenever possible I used a subset available in .NET which is `System.Windows.Forms.VisualStyles.ColorProperty`. Value `204 = Color` was missing though.

Reply:
Ah, found it - https://docs.microsoft.com/en-us/windows/win32/controls/property-typedefs
Though looking at the docs, these aren't all for colours...